pypi/azure-pipelines-pypi.yml

```diff
@@ -0,0 +1,113 @@
+
+jobs:
```

Comment:
Is there a way to only run the upload to pypi if we do a new release? I think PyPi has upload limits and we should not upload a new version on every commit.

Reply:
There should be, as long as we can detect creation of a tag or something else that marks the creation of a release. I always find I have to do a bit of Googling and playing around to get this to work, but there typically is a way. How are releases marked for Boolector? A tag? A branch?
That said, I've not detected any upload limits with PyPi on any of the projects I'm using it with.

Reply:
Releases are tagged, but there is also the releases page: https://github.com/Boolector/boolector/releases

If Azure doesn't have this kind of functionality we could also think of switching the PyPi upload to GitHub actions, which supports running workflows only for specific actions such as creating a release.